### DISCUSSION TITLE
miglior termostato smart

### DISCUSSION THREAD

**Discussion by federico89mi** [indomus/forum#382] [Hardware & Firmware]
Buongiorno a tutti,
volevo chiedere il vostro parere sui termostati smart ed eventualmente la compatibilità con le valvole per i caloriferi.
Per ora ho guardato solo il google nest termostato ma vedo che ce ne sono tanti altri.. La mia idea è comandare la temperatura della caldaia tramite termostato e home assistant e poter gestire la temperatura stanza per stanza con delle valvole wifi.
Quali kit consigliate?

grazie mille

**Comment by ilmaino1984** [comment]
Buongiorno,
Io uso da 4/5 anni termostati e valvole netatmo. Non mi hanno mai dato noie.
L'unica pecca è che l'integrazione con HA è via cloud.

**Reply by snakuzzo** [reply]
Io non uso personalmente un termostato, ma solo le valvole. Però potrei consigliarti Tado°.
E' possibile integrare il tutto anche senza l'integrazione nativa che sfrutta il cloud, ma con [Homekit Controller](https://www.home-assistant.io/integrations/homekit_controller/) (da non confondere con [Homekit](https://www.home-assistant.io/integrations/homekit/))...quindi localmente

**Reply by essegi65** [reply]
Io uso netatmo e francamente mi trovo altrettanto bene. Lo puoi usare sia con l'integrazione via cloud che con homekit controller in locale. Al momento non ho le valvole.

**Reply by federico89mi** [reply]
non capisco @snakuzzo come fai a essere senza termostato? Ah forse perchè non hai l'impianto autonomo?

**Reply by ilmaino1984** [reply]
> Io uso netatmo e francamente mi trovo altrettanto bene. Lo puoi usare sia con l'integrazione via cloud che con homekit controller in locale. Al momento non ho le valvole.

Effettivamente mi scordo sempre del componente Homekit. Funziona sia con Termostato che valvole

**Reply by snakuzzo** [reply]
> non capisco @snakuzzo come fai a essere senza termostato? Ah forse perchè non hai l'impianto autonomo?

Esatto...ho quello condominiale centralizzato :)

**Comment by KC-inDomus** [comment]
tado.

**Comment by mattapo** [comment]
Io quest'inverno ho provato un termostato diy con uno shelly 1 e firmware ESPhome e mi sono trovato benissimo; non l'ho mai toccato dalle temperature che ho preimpostato in base agli orari, peccato che non ho le termostatiche per avere una temperatura ottimale in tutte le stanze
